xbmc/video/VideoDatabase.cpp

```diff
@@ -4680,11 +4775,22 @@ void CVideoDatabase::UpdateTables(int iVersion)
       m_pDS->next();
     }
   }
+
+  if (iVersion < 101)
+  {
+    std::string strSQL = PrepareSQL("ALTER TABLE movie ADD c%02d integer", VIDEODB_ID_MOVIE_BOOKMARK);
```

Comment:
If you add it in the cXX fields gets automatically created. I created this pr some time ago so probably I should use a new field with a proper name like bookmarkFileId right? Probably I used a cXX field because the episode one is already one of them